src/integration-test/java/uk/gov/di/ipv/cri/passport/integrationtest/DataStoreIT.java

```diff
@@ -21,12 +25,13 @@
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
+@ExtendWith(SystemStubsExtension.class)
 public class DataStoreIT {
 
-    private static final String dynamodbUrl = "http://localhost:4567";
+    private static final String dynamodbUrl = "http://localhost:4566";
```

Comment:
Our intention was to take these from environment variables down the line which could now be set by your bash set-up script? I think making these env vars will make it more flexible by reducing the coupling between the test and the environment it needs to run in, e.g. just tell the test where the dynamodb endpoint is and off it goes.